user_guide/data_management_view.rst

```diff
@@ -0,0 +1,173 @@
+.. _user-guide-ui-components-view-pages:
+
+View Pages
+==========
+
+To see more details of a specific record, and to work directly with the record (create a task related to a customer, 
+appoint a calendar event for a user, turn a cart into an order, share a contact and so on), you need to get to its 
+View page.
+
+For example, this is a View page of a contact.
+
+|
+
+.. image:: ./img/data_management/view/view_01.png
+
+|
+
+On this View page you can see all the details of the record and location of the address on the map, can use the links 
+and action buttons as described in this article. 
+
+Get To a View Page
+------------------
+
+There are several ways to get to a View page:
+
+- Select the relevant record in a :ref:`grid <user-guide-ui-components-grids>` (click the |IcView| icon )
+
+- Find the records in the :ref:`Search Panel <user-guide-getting-started-search>`
+
+- Click :guilabel:`Save and Close` button on a :ref:`Create/Edit form <user-guide-ui-components-create-pages>`
+
+
+What You Can See on a View Page
+-------------------------------
+
+Details of the chosen record are displayed on a View page. (The list of properties shown is defined 
+by the system administrator with the  *Show On View* setting of the entity fields).
+
+Information in a view page is allocated in sections, so that you can easily switch from one section to another. All the 
+sections are placed one after another, so you can also scroll down to the required information. 
+
+For example, this is a View page of an opportunity with sections "General Information", "Activities" and 
+"Additional Information":
+
+|
+
+.. image:: ./img/data_management/view/view_page.png
+
+|
+
+Click the section name and this section will be displayed on the top of the page:
+
+      |
+
+  
+.. image:: ./img/data_management/view/view_page_tabs.png
+
+|
+
+.. _user-guide-ui-components-view-page-actions:
+
+What You Can Do from a View Page
+--------------------------------
+
+Manage the Record from a View Page
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Action buttons at the top right of each View page reflect the actions that you can do with the 
+record. 
+
+These include ability to share the record with other users (if applicable), to get to the 
+:ref:`Create/Edit form <user-guide-ui-components-create-pages>`, to delete a record and to perform different 
+:ref:`activities <user-guide-activities>` related to the record (make a task, create a calendar event, log a call, 
+write an email, etc.)
+
+  
+|
+
+.. image:: ./img/data_management/view/view_edit.png
+
+|
+
+
+If there are four or less actions available for the record, a separate button is created for each of them.
```

Comment:
Edit: If there are four or less actions available for a record, a separate button is created for each of them.